scripts/global_ens/stats/exevs_global_ens_headline_grid2grid_stats.sh

```diff
@@ -37,7 +37,33 @@ verify_type=$3
 #############################################################
 
 if  [ $ens = gefs ] || [ $ens = naefs ] || [ $ens = gfs ] ; then
-  if [ $verify_type = upper ] ; then 
+  if [ $verify_type = upper ] ; then
+
+   if [ $ens = gfs ] || [ [ $ens = gefs ] ; then
+    if [ ! -s ${COMIN}.${VDATE}/gefs/gfsanl.t00z.grid3.f000.grib2 ] ; then
+         	       
+       export subject="GFS analysis data missing for $ens headline stat job"
+       export maillist=${maillist:-'geoffrey.manikin@noaa.gov,binbin.zhou@noaa.gov'}
```

Comment:
@BinbinZhou-NOAA this line (46) needs to be removed now that the variable is exported from the ecf script.